api/app/src/main/resources/db/migration/V20240919__create_service_account.sql

```diff
@@ -0,0 +1,10 @@
+WITH inserted_user AS (
```

Comment:
I intentionally did not add a `service` role for the user. I don't think it is useful, and it opens the door to modifying that role through the API, which I'd rather avoid.

If we ever have a use case for it we can add it later as another migration.